lib/src/Pos/Launcher/Resource.hs

```diff
@@ -243,9 +244,9 @@ getRealLoggerConfig LoggingParams{..} = do
     overrideConsoleLog :: LoggerConfig -> LoggerConfig
     overrideConsoleLog = case lpConsoleLog of
         Nothing    -> identity
-        Just True  -> (<>) (defaultInteractiveConfiguration Info)
+        Just True  -> (<> defaultInteractiveConfiguration Info)
                       -- add output to the console with severity filter >= Info
-        Just False -> identity
+        Just False -> lcLoggerTree . ltHandlers %~ filter (not .  isWritingToConsole)
```

Comment:
I am pedantic, there is a superfluous space.
LGTM! I like your solution.

Reply:
Ho right :o ... I'll remove :+1: